client/src/containers/Profile/sagas.js

```diff
@@ -47,15 +47,12 @@ function* watchCurrentUserRequest() {
   yield takeEvery(fetchCurrentUser.TRIGGER, currentUserRequest);
 }
 
-function* loginGoogle({ payload: { user, history } }) {
+function* loginGoogle({ payload: { user } }) {
   try {
-    const { jwt } = user;
-    yield call(signupService.setToken, jwt);
     const authorizedUser = {
       id: user.id,
       username: user.username,
-      email: user.email,
-      token: jwt
+      email: user.email
```

Comment:
I think, will better:
```
const { id, username, email } = user;
const authorizedUser = { id, username, email }
```

Reply:
FIxed even further, there's no need to re-assemble that object now :)